ios/Runner/Info.plist

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>BGTaskSchedulerPermittedIdentifiers</key>
+	<array>
+        <string>task-identifier</string>
+        <string>be.tramckrijte.workmanagerExample.iOSBackgroundAppRefresh</string>
```

Comment:
I added Info.plist, just adding those 2 lines for the iOS setup. So the merge conflict resolution with #125 should be easy

Reply:
Theses two tasks does not exist in our code. Ours are named:
- scheduleDailyTasks
- trackingTask

See: https://github.com/fluttercommunity/flutter_workmanager/blob/d7e6ba5ef3796fafe54d9739e481f01058751b4d/example/ios/Runner/Info.plist#L5-L16